bin/eslint.js

```diff
@@ -4,4 +4,8 @@ const CODE_DIR = "/code";
 process.chdir(CODE_DIR);
 
 const ESLint = require("../lib/eslint");
-ESLint.run({ dir: CODE_DIR });
+const issues = ESLint.run({ dir: CODE_DIR });
```

Comment:
@wfleming There is a major difference here from the previous behaviour, as it now accumulates all issues before printing them all. I hope it is OK.

I see this could be potentially problematic on large repos and if the engine is actually making use of the output stream  (which I don't have context yet).